services/monitor/instances/monitor_gae_env_secrets/constraints/no_secrets_in_env_vars/constraint.yaml

```diff
@@ -0,0 +1,172 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPAppengineEnvSecretConstraintV1
+metadata:
+  name: no_secrets_in_env_vars
+  annotations:
+    description: AppEngine with ENV var containing suspicious keywords.
+    category: Secret Management
+spec:
+  severity: critical
+  match:
+    target: ["organization/"] # to_be_adapted
```

Comment:
Would you remove double quotes ?

Reply:
done

Reply:
I still see the double quotes, did you push your changes ?

Reply:
Yes, I think you're looking at an outdated version